File: 2025/04/20250408/32132.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int n;
    string str;

    ios_base::sync_with_stdio(false);
    cin.tie(nullptr);

    cin >> n;
    cin >> str;

    for (int i = 0; i < str.size() - 2; i++)
    {
        while (i + 2 < str.size() && (str.compare(i, 3, "PS4") == 0 || str.compare(i, 3, "PS5") == 0))
        {
            if (i + 3 < str.size())
                str = str.substr(0, i + 2) + str.substr(i + 3);
            else
                str = str.substr(0, i + 2);
        }
    }

    cout << str << "\n";

    return 0;
}

File: 2025/04/20250408/32132.py
n = int(input())
string = input()

for i in range(len(string)):
    while i + 3 <= len(string) and (string[i:i+3] == 'PS4' or string[i:i+3] == 'PS5'):
        if i + 3 < len(string):
            string = string[:i+2] + string[i+3:]
        else:
            string = string[:i+2]
print(string)